README.md

```diff
@@ -53,7 +53,7 @@ services:
       - NET_ADMIN # Required if you are using Pi-hole as your DHCP server, else not needed
     restart: unless-stopped
 ```
-2. Run `docker-compose up -d` to build and start pi-hole
+2. Run `docker compose -f docker-compose.yml up` to build and start pi-hole
```

Comment:
Is there no `-d` in the `docker compose` command?

Reply:
@PromoFaux actually there is.